src/app/cube/new-home/help/components/help-card/help-card.component.ts

```diff
@@ -7,7 +7,7 @@ import { Component, Input, OnInit } from '@angular/core';
 })
 export class HelpCardComponent implements OnInit {
 
-  @Input() option: {title: string, description: string, icon: string, iconColor: string};
+  @Input() option: {title: string, description: string, icon: string, iconColor: string, link?: string[]};
```

Comment:
I don't think link should be a string array.

Reply:
I made it that way so that i could use it as is in the routerlink. what would you recommend?